### DISCUSSION TITLE
Dynamic Model Changes

### DISCUSSION THREAD

**Discussion by phiferch** [LostRuins/lite.koboldai.net#56] [Ideas]
Are there any plans to add the ability to Dynamically Change the Model from within the UI? I typically access this remotely through a VPN I have running, and it works great, but changing the model requires me to remote into the machine I have it running on, kill the application, and then reload it with a new model.

**Comment by LostRuins** [comment]
No plans for this currently - releasing and reallocating the memory over the UI would be rather tricky to setup. 

An alternative may be using a docker container and controlling the container restarts via a web frontend.
